### PR TITLE
Calculator UI

### PR DESCRIPTION
**Pull request by JakariaJishan**
- [x] Use functional components
- [x] Design Calculator

### CODE REVIEW

File: src/components/calculator/calculator.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+body {
+  color: white;
+}
+
+.wrapper {
+  /* display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center; */
```
Comment (AmaduKamara) [inline src/components/calculator/calculator.css]: - [x] Let's try not to have commented code in our codebase as they make our code dirty and look unprofessional. It's a good convention to have a clean and professional codebase free from commented or unused codes. Kindly remove them 👍🏽

File: src/components/calculator/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      <div className="minus">-</div>
+      <div className="one">1</div>
+      <div className="two">2</div>
+      <div className="three">3</div>
+      <div className="plus">+</div>
+      <div className="zero">0</div>
+      <div className="dot">.</div>
+      <div className="equal">=</div>
+    </div>
+  </div>
```
Comment (AmaduKamara) [inline src/components/calculator/Calculator.js]: I really love the way you styled your app, it's cool and very pleasing, but; 
- [x] According to the design guidelines, you're required to follow the design given and to match your design with that. Please modify your styles to match the original design. The goal is to train you to work on design guidelines given for a project as that's what awaits you in your future job. 👍🏽
 
Screenshot 2023-02-21 at 8 53 18 PM

File: src/components/calculator/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+const Calculator = () => (
+  <div className="wrapper">
+    <div className="output">
+      0
+    </div>
+    <div className="keys">
+      <div className="ac">AC</div>
+      <div className="plus-minus">+/-</div>
+      <div className="percent">%</div>
+      <div className="division">/</div>
```
Comment (AmaduKamara) [inline src/components/calculator/Calculator.js]: - [x] For the division operator, you're to use the other sign and not the forward slash. You can use the HTML entity code `&divide;` to achieve it or you copy the sign from the project requirement and past it in your JSX 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️ 

Hi @JakariaJishan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used functional components 👍🏽
✅ The app runs with no error 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [x] You're required to clear and remove all boilerplate codes and files generated by create-react-app, images, logos, and styles. Kindly see them highlighted in the screenshot below:

Screenshot 2023-02-21 at 8 48 41 PM


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Owusu-Desmond — APPROVED** [review]
Hi @JakariaJishan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**